browser/src/control/jsdialog/Widget.ColorPicker.ts

```diff
@@ -445,6 +293,7 @@ function createPaletteSwitch(
 			event.preventDefault(); // Prevent default behavior
```

Comment:
above keyboard event listener also could be moved to util (it doesn't use widget-specific things, just our helper functions)